pytest-embedded/pytest_embedded/plugin.py

```diff
@@ -1253,6 +1254,26 @@ def _duplicate_items(items: t.List[_T]) -> t.List[_T]:
 
     @pytest.hookimpl(hookwrapper=True, trylast=True)
     def pytest_collection_modifyitems(self, items: t.List[Function]):
+
+        for item in items:
+            skip_marker = item.get_closest_marker('skip_if_soc')
+            if not skip_marker:
+                continue
+
+            if not hasattr(item, 'callspec'):
+                logging.warning(
+                    f"Test item '{item.originalname}'. "
+                    f"Ensure 'target' is included in the test @pytest.mark.parametrize when using 'skip_if_soc'."
+                )
+                continue
```

Comment:
what about we support `target` from CLI as well (not "auto")?

so it could also be used as `skipif` when user call `pytest --target esp32`

Reply:
Agree, added